roles/services/crawler/templates/config.yml

```diff
@@ -9,11 +9,16 @@ opensearch:
   bulk_getter_batch_size: 48                          # Item treshold for execution of bulk gets.
   bulk_getter_batch_timeout: 1s                       # Time treshold for bulk gets.
 redis:
-    addresses:
-{% for host in groups['redis'] %}
-      - {{ hostvars[host].ansible_all_ipv4_addresses | ipaddr('private') | ipaddr('address') | first }}:7000
-      - {{ hostvars[host].ansible_all_ipv4_addresses | ipaddr('private') | ipaddr('address') | first }}:7001
-{% endfor %}
+  addresses:
+  # struggled with that part
+  # at first there was an error regarding 'redis' group does not exist
+  # but later 'ansible.vars.hostvars.HostVarsVars object' has no attribute 'ansible_all_ipv4_addresses'
+    - '172.16.0.2:7000' # redis-1
+    - '172.16.0.2:7001' # redis-1
+    - '172.16.0.3:7000' # redis-2
+    - '172.16.0.3:7001' # redis-2
+    - '172.16.0.4:7000' # redis-3
+    - '172.16.0.4:7001' # redis-3
```

Comment:
it is a temp solution, I just wanted to make sure that the whole cycle is working. Let's discuss it as soon as you're available

Reply:
In the playbook, `gather_facts` needs to be called on the redis hosts. In addition, as some hosts failing should not cause the crawler deployment to fail in addition, we should check in the loop whether this variable is set.

Please see https://github.com/ipfs-search/ipfs-search-deployment/pull/34/files#diff-c8fd42263abb143537dc06f2d96d6ee550458e05a97c3b208f3357a38e6bb896R2 as an example of this.